pkg/channel/consolidated/reconciler/controller/controller.go

```diff
@@ -125,5 +147,24 @@ func NewController(
 		Handler:    controller.HandleAll(grCh),
 	})
 
+	podInformer.Informer().AddEventHandler(cache.FilteringResourceEventHandler{
```

Comment:
Given the lack of e2e currently, this logical group in particular might be good to add unit tests to

Reply:
do you mind elaborating more? I'm not sure exactly what should such unit tests do? when pods are deleted, that `CancelPodProbing` gets called?

Reply:
you can inject objects into the fake informers, and ensure that the filterfunc's are behaving as expected

Reply:
I think that's pretty much covered here? https://github.com/knative/pkg/blob/main/reconciler/filter_test.go

Reply:
I'm talking about the specific number and type of objects we'll have before, and then after a resync given the number of specific handlers and filterfuncs we have.  This part of the diff has a lot of codecov warnings about lack of unit tests.  Is there a particular reason we're looking to avoid sufficient coverage for this block?

Reply:
sounds reasonable to get them covered via tests. if possible.